Riot.xcodeproj/project.pbxproj

```diff
@@ -6981,7 +6973,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = au.gov.act.health.aether.shareExtension;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE = "1a3be143-50c7-4ae2-834e-00596a053141";
-				PROVISIONING_PROFILE_SPECIFIER = "";
+				PROVISIONING_PROFILE_SPECIFIER = "au gov act health aether shareExtension Distributi";
```

Comment:
Is Distribution intentionally truncated?

Reply:
Yes unfortunately - Apple restricts the plain text description for provisioning profiles to 50 characters (for some reason).  We stuck to the same naming convention used for earlier profiles so that meant with the longer target names the Distribution part gets truncated.